src/components/Text/utils.ts

```diff
@@ -3,17 +3,15 @@ import { TextVariants } from './types';
 
 export const determineTextClassName = (variant: TextVariants = TextVariants.body1): string => {
   // our javascript const doesn't exactly match the tokens
-  const tokenVariant = variant.replace('ctaSm', 'cta-sm').replace('ctaLg', 'cta-lg');
 
-  return `${px}-text--${tokenVariant.charAt(0).toLowerCase() + tokenVariant.slice(1)}`;
+  return `${px}-text--${variant.charAt(0).toLowerCase() + variant.slice(1)}`;
 };
 
 export const determineDefaultTextElement = (variant: TextVariants = TextVariants.body1): string => {
-  if ([TextVariants.display0, TextVariants.display1].includes(variant)) return 'h1';
-  if ([TextVariants.display2, TextVariants.display3, TextVariants.display4].includes(variant)) return 'h3';
   if (variant === TextVariants.blockquote) return 'blockquote';
-
   if (variant.includes('body')) return 'p';
+  if (variant.includes('string')) return 'span';
+  if (variant.includes('title')) return 'span';
```

Comment:
seems like a strange choice for title? Is this on purpose?

Reply:
@davidicus as we discussed, we're not sure where to use `title` yet